SubMotionVR Mod/SubMotion/SubMotion/Patchers/ArmsController.cs

```diff
@@ -152,103 +88,48 @@ public static void Postfix(ArmsController __instance)
                 return;
             }
 
+            PDA pda = VRHandsController.main.pda;
+            Player player = VRHandsController.main.player;
             if ((Player.main.motorMode != Player.MotorMode.Vehicle && !player.cinematicModeActive) || pda.isActiveAndEnabled)

```

Comment:
it might be better to move this logic into the controller class and only call `UpdateHandPositions` in the patch method